lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -91,6 +103,7 @@ class RenderBlockingResources extends Audit {
       node.traverse(node => deferredNodeIds.add(node.id));
 
       // "wastedMs" is the download time of the network request, responseReceived - requestSent
+      // @ts-ignore - TODO(phulce): nodeTiming.startTime/endTime shouldn't be optional by this point?
```

Comment:
`nodeTimings` is going to need something to differentiate between the in progress object with properties still being defined (and therefore optional) vs what's exposed externally where presumably most (all?) properties are defined. I added @patrickhulce here but I can also take it eventually :)

Reply:
heh, didn't see this but yeah bingo